refinery/tool_manager/utils.py

```diff
@@ -283,19 +280,17 @@ def create_tool_analysis(validated_analysis_config):
     user = common_analysis_objects["user"]
 
     try:
-        tool = Tool.objects.get(
+        tool = WorkflowTool.objects.get(
             uuid=validated_analysis_config["toolUuid"]
         )
-    except (Tool.DoesNotExist, Tool.MultipleObjectsReturned) as e:
+    except (WorkflowTool.DoesNotExist,
+            WorkflowTool.MultipleObjectsReturned) as e:
         raise RuntimeError("Couldn't fetch Tool from UUID: {}".format(e))
 
     analysis = AnalysisFactory(
         uuid=str(uuid.uuid4()),
         summary="Galaxy workflow execution for: {}".format(tool.name),
-        name="{} - {}".format(
-            name,
-            get_aware_local_time().strftime("%Y-%m-%d @ %H:%M:%S")
-        ),
+        name="{} - {}".format(name, tool.uuid),
```

Comment:
The point of this is to get a human-friendly name? I'm not sure we want to include the whole uuid still. Imagining what the facet will look like. Sorry if I'm thinking about this wrong.